backend/src/graphql/resolvers/UserResolver.ts

```diff
@@ -2,12 +2,14 @@
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 
 import { Resolver, Query, Args, Arg, Authorized, Ctx, UseMiddleware, Mutation } from 'type-graphql'
+import { from_hex as fromHex } from 'libsodium-wrappers'
```

Comment:
Why is this needed? Please explain.

Reply:
For translating the pubkey from hex format into binary format.
I choose libsodium because I also need it later for check signatures and translate base64 to binary.
Login-Server use libsodium for every crypto-stuff and also for base64 and hex encoding. In the past I had difficulties with native base64 encoding on php so I used also libsodium there.